### ISSUE TITLE
Orders (and Subscriptions) created manually via WP Admin have incorrect `prices_include_tax` value set

### ISSUE THREAD

**Issue by mattallan** [woocommerce/woocommerce#51426]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Manually creating an Order via the **WooCommerce > Orders > Add Order** creates an order with an incorrect `prices_include_tax` value that does not match the stores tax settings.

This issue has bigger impact for stores using subscriptions because when this subscription renews and customers go to pay for the pending renewal order via the checkout, the checkout totals don't properly include the tax:

Edit Order screen (has correct totals):

![image](https://github.com/user-attachments/assets/80e920be-5c2e-44d6-9fef-b1fb2ea9d791)

Paying for this pending renewal order via the checkout (incorrect totals not including tax correctly :x:):

![image](https://github.com/user-attachments/assets/95be6e7b-9e24-49a8-bc18-8c419b87af0c)

I created an issue for this on WooCommerce Subscriptions (https://github.com/Automattic/woocommerce-subscriptions-core/issues/682) but I think a fix within WooCommerce Core makes more sense.

### Expected behavior

When stores have their tax settings set to "I will enter prices inclusive of tax", we should be creating subscriptions with the correct value set for prices_include_tax that match the stores settings.

### Actual behavior

On my store with tax settings set to "I will enter prices inclusive of tax", Orders and Subscriptions that are created via WP Admin have `0` set for `prices_include_tax`:

![image](https://github.com/user-attachments/assets/de9a8005-1d0b-4323-ada1-1842a9580675)

### Steps to reproduce

1. Go to **WooCommerce > Settings > Tax**
2. Set "Prices entered with tax" to "Yes"
3. Go to **WooCommerce > Orders > Add Order** and create a pending order
4. Query the wc_order_operational_data table for this new Order ID and confirm that `prices_include_tax` is set to 0.

### WordPress Environment

.

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by juliaamosova** [comment]
Hi @mattallan! Thanks for reporting the issue. Do you know if this is a new issue (e.g. started happening after WooCommerce 9.3 release) or if it has been present for a while?

**Comment by mattallan** [comment]
Hi @juliaamosova, this issue has been present for a long while, maybe even forever 😞 

To clarify, this issue doesn't seem to impact regular WC Orders because they go through the Pay for Order flow. However, in WooCommerce Subscriptions we use the checkout flow to pay for pending/failed renewal orders which is where this issue exists.

The main concern with this bug is merchants can't easily see the problem as the order items look fine. It's not until the customer pays for the order manually via the checkout and merchants notice they didn't get charged the correct amount. It's an even bigger problem when merchants have migrated 1000s of subscriptions (which happened in one case).

---

I've pushed up a fix in WooCommerce core here #51428 and I'd love to get feedback from your team as to whether you think this should be fixed in core, or fix it in WooCommerce Subscriptions since that's where the bug exists.

Thanks!

**Comment by dennisnissle** [comment]
This is a serious issue and leaves all manually created orders in an invalid (tax) state. Very bad that there is no fix available for over 6 month now. @mikejolley could you please escalate the issue?

**Comment by mikejolley** [comment]
cc @tjcafferkey There is an old PR for this that was never finalised.

**Comment by tjcafferkey** [comment]
I'll look into the existing work that was done around this issue, this week. If not much effort is required to get this into a merge-able state then I don't see a reason why we can't proceed with it.

**Comment by flakas** [comment]
Hi @mattallan - thanks for raising this.

I know this has been open a while, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

It looks like this might be covered by the following work: https://github.com/woocommerce/woocommerce/pull/51428. I've linked your issue to it so you can follow the progress there.

Marking it for engineering review.

Where it's at: Leave linked to PR #51428 and have order-management engineering confirm the admin new-order path is the remaining affected creation flow.
